Pipfile

```diff
@@ -0,0 +1,30 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+black = "*"
+pytest = "*"
+six = "*"
+codecov = "*"
+"flake8" = "*"
+pytest-cov = "*"
+neovim = "*"
+pytest-sugar = "*"
+pytest-clarity = "*"
+autopep8 = "*"
+gspread =  "*"
+oauth2client = "*"
+
+[packages]
+pathlib = "*"
+py = "*"
+django = "*"
+
+[requires]
+python_version = "3.6"
```

Comment:
Should python 3.6 be explicitly required?

Reply:
@ilikerustoo

Reply:
We deleted this but it might have accidentally come back in the merging process. Also for a sure check for errors, we should be looking at `Travis` instead of individual machines.